src/app/layout.tsx

```diff
@@ -25,11 +27,23 @@ export default async function RootLayout({
 }: Readonly<{
     children: ReactNode
 }>) {
+    const clerkPublishableKey = process.env.NEXT_PUBLIC_CLERK_PUBLISHABLE_KEY
+    if (!clerkPublishableKey) return <ErrorAlert error={'missing clerk key'} />
+
     return (
-        <html lang="en">
-            <body>
-                <Providers>{children}</Providers>
-            </body>
-        </html>
+        <ClerkProvider
+            publishableKey={clerkPublishableKey}
+            localization={{
+                organizationSwitcher: {
+                    personalWorkspace: 'Researcher Account',
+                },
+            }}
+        >
+            <html lang="en">
+                <body>
+                    <Providers>{children}</Providers>
+                </body>
+            </html>
+        </ClerkProvider>
```

Comment:
this is what we'd discussed, unfortunately we can't use Clerk on pre-rendered pages like 404 and not found